spec/models/emotion_spec.rb

```diff
@@ -47,16 +47,24 @@
   describe 'Validations' do
     it { is_expected.to validate_presence_of :word }
     it { is_expected.to validate_length_of(:word).is_at_least(2).is_at_most(15) }
-    it { is_expected.to define_enum_for(:category).with_values(negative: 0, neutral: 1, positive: 2) }
 
-    it 'the same emotion word should not be used' do
-      new_word = FactoryBot.build(:emotion, word: emotion.word)
-      expect(new_word).to_not be_valid
+    context 'uniqueness of word within category' do
+      let!(:existing_emotion) { create(:emotion, word: 'pressured', category: :positive) }
+
+      it 'does not allow the same emotion word with the same category' do
+        new_emotion = FactoryBot.build(:emotion, word: existing_emotion.word, category: existing_emotion.category)
+        expect(new_emotion).to_not be_valid
+      end
+
+      it 'allows the same emotion word with a different category' do
+        new_emotion = FactoryBot.build(:emotion, word: existing_emotion.word, category: :negative)
+        expect(new_emotion).to be_valid
+      end
     end
 
     it 'the capitalized emotion word should not be used' do
-      new_word = FactoryBot.build(:emotion, word: emotion.word.upcase!)
-      expect(new_word).to_not be_valid
+      new_word = FactoryBot.build(:emotion, word: Faker::Emotion.unique.adjective)
+      expect(new_word).to be_valid
```

Comment:
@Maryna-Harasko you change the meaning of this test, do you still want to check `the capitalized emotion word should not be use`? If yes please add `upcase!`